Library/Formula/telegram.rb

```diff
@@ -0,0 +1,40 @@
+class Telegram < Formula
+  homepage "https://github.com/vysheng/tg"
+  url "https://github.com/vysheng/tg/archive/1.0.5.1.tar.gz"
+  sha1 "72f6a9f89ee9db90b573bad25e567a2015115848"
+
+  depends_on "openssl"
+  depends_on "libevent"
+  depends_on "libconfig"
+  depends_on "readline"
+  depends_on "lua"
+
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!